alerter/src/monitors/node/cosmos.py

```diff
@@ -882,6 +898,7 @@ def _process_retrieved_tendermint_rpc_data(self, data: Dict) -> Dict:
                     'node_id': self.node_config.node_id,
                     'node_parent_id': self.node_config.parent_id,
                     'time': datetime.now().timestamp(),
+                    'is_mev_tendermint_node': 'is_peered_with_sentinel' in data,
```

Comment:
Just to clarify: `is_mev_tendermint_node` is automatically derived via API/RPC calls, and in **no way meant to be configurable** by the node operator in the frontend?

Reply:
If so, you can ignore this comment.